server/src/main/java/org/spine3/server/validate/BooleanFieldValidator.java

```diff
@@ -48,9 +48,13 @@
         super(descriptor, fieldValues, rootFieldPath, false);
     }
 
+    /**
+     *  Boolean field can't be determined if it set or not, because protobuf 'false' value and 'no value' are the same
```

Comment:
Please have dot at the end of the sentence. While we are here, I'd say that it would be better to say that
”In Protobuf there is no way to tell if the value is {@code false} or was not set.”